modules/asg/main.tf

```diff
@@ -28,6 +28,9 @@ locals {
   default_eni_sg_ids       = flatten([for k, v in var.interfaces : v.security_group_ids if v.device_index == 0])
   default_eni_public_ip    = flatten([for k, v in var.interfaces : v.create_public_ip if v.device_index == 0])
   account_id               = data.aws_caller_identity.current.account_id
+  // this is done in case you store it in a hierarchy. if you just provide a name appennd a forward slash
+  delicense_param = startswith(coalesce(var.delicense_ssm_param_name, ""), "/") ? var.delicense_ssm_param_name : "/${var.delicense_ssm_param_name}"
```

Comment:
Unfortunately this still doesn't work - please see error in 
https://github.com/PaloAltoNetworks/terraform-aws-swfw-modules/actions/runs/7873195387/job/21480004195#step:14:4638 :

```
          	Error:      	Received unexpected error:
          	            	FatalError{Underlying: error while running command: exit status 1; ╷
          	            	│ Error: Error in function call
          	            	│ 
          	            	│   on ../../modules/asg/main.tf line 32, in locals:
          	            	│   32:   delicense_param = startswith(coalesce(var.delicense_ssm_param_name, ""), "/") ? var.delicense_ssm_param_name : "/${var.delicense_ssm_param_name}"
          	            	│     ├────────────────
          	            	│     │ while calling coalesce(vals...)
          	            	│     │ var.delicense_ssm_param_name is null
          	            	│ 
          	            	│ Call to function "coalesce" failed: no non-null, non-empty-string
          	            	│ arguments.
          	            	╵}
          	Test:       	TestIdempotence
```

I propose below solution:

```suggestion
  delicense_param = try(startswith(var.delicense_ssm_param_name, "/") ? var.delicense_ssm_param_name : "/${var.delicense_ssm_param_name}", null)
```